src/decls/Stripe.js

```diff
@@ -25,4 +25,14 @@ declare type StripeShape = {
     type: string | ElementShape,
     options: mixed
   ) => Promise<{token?: MixedObject, error?: MixedObject}>,
+  createPaymentMethod: (
+    type: string,
+    element: ElementShape,
+    data: mixed
+  ) => Promise<{paymentMethod?: MixedObject, error?: MixedObject}>,
+  handleCardPayment: (
+    clientSecret: string,
+    element: ElementShape | MixedObject,
+    options: mixed
+  ) => Promise<{paymentIntent?: MixedObject, error?: MixedObject}>,
```

Comment:
Have we filed follow-up tasks to deal with `confirmPaymentIntent` and `handleCardAction`?

Reply:
`handleCardAction` will not require any work in RSE. We could all the types though. `confirmPaymentIntent` will be tricky, will add a follow up.